ARCs/arc-0200.md

```diff
@@ -189,14 +189,18 @@ A smart contract token that is compliant with this standard MUST implement the f
 
 Ownership of a token by a zero address indicates that a token is out of circulation indefinitely, or otherwise burned or destroyed.
 
-The methods `arc200_transfer` and `arc200_transferFrom` method MUST error when the balance of `from` is insufficient. In the case of the `arc200_transfer` method, from is implied as the `owner` of the token.
-The `arc200_transferFrom` method MUST error unless called by an `spender` approved by an `owner`.
+The methods `arc200_transfer` and `arc200_transferFrom` method MAY error when the balance of `from` is insufficient. In the case of the `arc200_transfer` method, from is implied as the `owner` of the token. 
+The `arc200_transferFrom` method MAY error unless called by an `spender` approved by an `owner`.
 The methods `arc200_transfer` and `arc200_transferFrom` MUST emit a `Transfer` event.
 A `arc200_Transfer` event SHOULD be emitted, with `from` being the zero address, when a token is minted.
 A `arc200_Transfer` event SHOULD be emitted, with `to` being the zero address, when a token is destroyed.
 
 The `arc200_Approval` event MUST be emitted when an `arc200_approve` or `arc200_transferFrom` method is called successfully.
 
+The methods `arc200_transfer` `arc200_transferFrom`, and `arc200_approve` SHOULD return byte `1` on success and byte `0` on failure.
+
+The method `arc200_decimals` SHOULD return a value less than or greater than uint64 `256`
```

Comment:
I don't understand this line. Why should it be Less OR Greater than uint64 `256`?